.travis.yml

```diff
@@ -306,6 +310,12 @@ before_install:
         # this assumes TEST_THEANO runs in the same build as TEST_MATPLOTLIB
         conda install theano;
     fi
+
+  - if [[ "${TEST_SYMENGINE}" == "true" ]]; then
+        conda config --add channels conda-forge;
+        conda config --add channels symengine;
```

Comment:
SymEngine channel is updated by this, https://github.com/symengine/python-symengine-feedstock/tree/symengine-tokens. I change the commit number and the bump the build number to update it.

We can use a git tag of symengine and build and use that version from `conda-forge` channel. I've had to make lots of changes to python wrappers and therefore I'm not sure we want to make a tag everytime I do a change to the python wrappers for sympy compatibility.

We can always do a symengine release for each sympy release.